src/components/icon/icon.stories.mdx

```diff
@@ -115,6 +117,34 @@ const MyComponent = () => (
   </Story>
 </Preview>
 
+### Custom colors
+It's possible to change the color of the Icon by using the `color` prop or background color by `bg` prop.
+These props will override `iconColor` and `bgTheme` prop values.
+See <LinkTo kind='Documentation/Colors' name="page">Colors</LinkTo> for more information.
+
+<Preview>
+  <Story name="Custom colors" parameters={{ info: { disable: true }, chromatic: { disable: true }}}>
```

Comment:
Is this story supposed to be disabled in chromatic? Might be useful to have snapshots of these props working

Reply:
I think we need to have it in `All` story as `visual` part

Reply:
@sjohnson-sage I've added these to `All` story

Reply:
Thanks